jenkins/Jenkinsfile-blossom.premerge

```diff
@@ -230,6 +232,78 @@ pipeline {
                 }
             }
         } // end of Premerge Test
+
+        // TODO: support parallel testing for different spark versions
+        stage('Databricks Test') {
+            when {
+                beforeAgent true
+                expression {
+                    !skipped
+                }
+            }
+
+            agent {
+                kubernetes {
+                    label "premerge-test-${BUILD_TAG}"
+                    cloud 'sc-ipp-blossom-prod'
+                    yaml "$pluginPremerge"
+                    workspaceVolume persistentVolumeClaimWorkspaceVolume(claimName: "${PVC}", readOnly: false)
+                    customWorkspace "${CUSTOM_WORKSPACE}"
+                }
+            }
+
+            environment {
+                DATABRICKS_TOKEN = credentials("SPARK_DATABRICKS_TOKEN")
+                DATABRICKS_SECRET = credentials("SPARK_DATABRICKS_SECRET")
+                BUILD_CLUSTER_ID = '1015-114738-tilde12'
+                TEST_CLUSTER_ID = '0916-144915-ulcer295'
+                SPARK_VERSION = '3.0.0-databricks'
+                CI_RAPIDS_JAR = 'rapids-4-spark_2.12-0.1-SNAPSHOT-ci.jar'
+                CI_CUDF_JAR = 'cudf-0.14-cuda10-1.jar'
+            }
+
+            steps {
+                script {
+                    container('plugin-premerge') {
+                        def VER_INFO = sh(returnStdout: true,
+                            script: '. jenkins/version-def.sh>&2 && echo -n "CUDA_CLASSIFIER=$CUDA_CLASSIFIER;SCALA_VERSION=$SCALA_BINARY_VER;CUDF_VERSION=$CUDF_VER;DATABRICKS_VERSION=$PROJECT_VER;"')
+                        def CUDA_CLASSIFIER=subString(VER_INFO, 'CUDA_CLASSIFIER=', ';')
+                        def SCALA_VERSION=subString(VER_INFO, 'SCALA_VERSION=', ';')
+                        def CUDF_VERSION=subString(VER_INFO, 'CUDF_VERSION=', ';')
+                        def DATABRICKS_VERSION=subString(VER_INFO, 'DATABRICKS_VERSION=', ';')
```

Comment:
can we do sth simple like just split by ";" and "," ? func `subString` does not look like that cool